app/build.gradle

```diff
@@ -20,13 +20,14 @@ android {
   }
 }
 
+
```

Comment:
We had some spacing issues which we resolved with auto format, so these attributes just moved around a bit.